docs/swagger.json

```diff
@@ -525,6 +525,9 @@
         "/register": {
             "post": {
                 "responses": {
+                    "400": {
+                        "description": "Validation error."
+                    },
```

Comment:
I don't think that you need to add the response here as well 🤔

Reply:
ok, will remove this